xdsl/interactive/app.py

```diff
@@ -421,18 +439,32 @@ def update_output_operation_count_tuple(self) -> None:
             case Exception():
                 self.output_operation_count_tuple = ()
             case ModuleOp():
+                # sort tuples alphabetically by operation name
                 self.output_operation_count_tuple = tuple(
-                    count_number_of_operations(self.current_module).items()
+                    (k, v)
+                    for (k, v) in sorted(
+                        count_number_of_operations(self.current_module).items()
+                    )
                 )
 
-    def watch_output_operation_count_tuple(self) -> None:
+        self.update_operation_count_diff_tuple()
+
+    def update_operation_count_diff_tuple(self) -> None:
```

Comment:
Can we rely on the reactive infrastructure and have this be called automatically?
```suggestion
    def compute_operation_count_diff_tuple(self) -> None:
```

Reply:
There's a risk of it being called twice, once when the input updates, and again when the output updates, it would be good to figure out whether textual is smart enough not to do that

Reply:
I don't want it to be called unnecessarily though... compute functions fire whenever ANY reactive variable is fired.. should I still do it

Reply:
Good to know, thank you